pkg/cluster/cluster.go

```diff
@@ -427,11 +427,17 @@ func populateClusterFieldsFromLabels(cluster *k3d.Cluster) error {
 
 		// get image volume // TODO: enable external image volumes the same way we do it with networks
 		if cluster.ImageVolume == "" {
-			if imageVolumeName, ok := node.Labels["k3d.cluster.imageVolume"]; ok {
+			if imageVolumeName, ok := node.Labels[k3d.ImageVolumeLabelName]; ok {
 				cluster.ImageVolume = imageVolumeName
 			}
 		}
 
+		// get k3s cluster's secret
+		if cluster.Secret == "" {
```

Comment:
Let's rename this to `Token` instead of `secret`? WDYT?

Reply:
In commit 18811fe i try by renaming all 'secret' to token even into `k3d create cluster` command. Maybe it is to much ?

Reply:
Since we're still in `beta`, I guess it's fine :+1: